README.md

```diff
@@ -70,16 +70,39 @@ steps:
     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+##### Interacting with another Repository
+
 Note that the default GITHUB_TOKEN will only work for API operations within _the
 same repo that runs this workflow_. If you need to access or write to other
 repositories, [generate a Personal Access Token][pat] with `repo` scope and add
 it to your [repository secrets][].
 
-
 [github actions]: https://help.github.com/en/actions/automating-your-workflow-with-github-actions
 [pat]: https://github.com/settings/tokens
 [repository secrets]: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets
 
+Additionally, in order to use git with this external repository you will need to make additional configuration
+changes to your actions execution enviroment.
+
+```bash
+# Configure hub to use https instead of the git protocol
+git config --global hub.protocol https
```

Comment:
For which hub command is this necessary? I'm guessing `hub clone`/`hub remote add`?